### DISCUSSION TITLE
Custom border widths breaks default border width

### DISCUSSION THREAD

**Discussion by completewebco** [tailpress/tailpress#40] [Q&A]
For some reason when I add custom border widths to tailwind.config.js, the default border-with: 1px gets ignored. Why is this?

borderWidth: {
            default: '1px',
            '0': '0',
            '2': '2px',
            '3': '3px',
            '4': '4px',
            '6': '6px',
            '8': '8px',
            '10': '10px',
            '20': '20px',

        },

**Comment by jeffreyvr** [comment]
Default must be written in caps.

```
DEFAULT: '1px',
```

https://tailwindcss.com/docs/border-width#border-widths
